app/controllers/games_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
+
+  def new
+    @game = Game.new
+  end
+
+  def create
+    @game = Game.create!(game_params.merge(white_player_id: current_user.id))
+    if @game.save
+      redirect_to game_path(@game)
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def show
+    begin 
+      @game = Game.find(params[:id])
+    rescue
+      if @game.nil?
```

Comment:
Generally speaking, you should not use `rescue` without providing a specific exception class. Rails and Ruby both use exceptions to communicate a wide range of failures, so a blanket `rescue` could result in strange behavior and lots of debugging issues by masking all sorts of problems.

The particular exception class you're looking for here is `ActiveRecord::RecordNotFound` (http://guides.rubyonrails.org/active_record_querying.html#retrieving-a-single-object) and the syntax is just `rescue ActiveRecord::RecordNotFound`. This approach also has the added benefit of making the conditional on the next line unnecessary since `render` will now be called only when the game cannot be found.